router/pkg/config/config.go

```diff
@@ -66,17 +65,32 @@ type ResponseTraceHeader struct {
 	HeaderName string `yaml:"header_name" envDefault:"x-wg-trace-id"`
 }
 
+type SubgraphTracingAttribute struct {
+	Key        string `yaml:"key"`
+	Expression string `yaml:"expression"`
+}
+
+type SubgraphTracing struct {
+	Attributes []SubgraphTracingAttribute `yaml:"attributes"`
+}
+
 type Tracing struct {
 	Enabled             bool                `yaml:"enabled" envDefault:"true" env:"TRACING_ENABLED"`
 	SamplingRate        float64             `yaml:"sampling_rate" envDefault:"1" env:"TRACING_SAMPLING_RATE"`
 	ParentBasedSampler  bool                `yaml:"parent_based_sampler" envDefault:"true" env:"TRACING_PARENT_BASED_SAMPLER"`
 	Exporters           []TracingExporter   `yaml:"exporters"`
 	Propagation         PropagationConfig   `yaml:"propagation"`
 	ResponseTraceHeader ResponseTraceHeader `yaml:"response_trace_id"`
+	Subgraph            SubgraphTracing     `yaml:"subgraph"`
 
 	TracingGlobalFeatures `yaml:",inline"`
 }
 
+type RequestTransportTrace struct {
```

Comment:
We don't need this anymore, however we would still want to look at either adding a "Enabling" flag, or if no one has referenced "subgrpah.trace" we can auto disable the tracing. Favouring the latter.